src/lib/parseArgs/parseTest.cpp

```diff
@@ -104,7 +104,7 @@ PaArgument paArgs[] =
   { "-f1",      &f1,     "F",      PaFloat,  PaOpt, _i 3.1,   0,       100,     "float"               },
   { "-d1",      &d1,     "D",      PaDouble, PaOpt, _i 3.3,   0,       100,     "double"              },
   { "--lmTest", &lmTest, "",       PaBool,   PaOpt, FALSE,    FALSE,   TRUE,    "test logMsg"         },
-  { NULL,       &v2,     "V2",     PaInt,    PaOpt, 19,       PaNL,    1000,    "env var 2"           },
+  { NULL,       &v2,     2,     PaInt,    PaOpt, 19,       PaNL,    1000,    "env var 2"           },
```

Comment:
Most of the "v2" usage in this kind of structures (specially in the URL routes for service routines) has been correctly ignored by the replace... however, it seems I miss this.

Fixing soon.